test/internal/modules/agreements/controller.js

```diff
@@ -22,11 +22,18 @@ const createRequest = () => ({
   params: {
     agreementId: 'test-agreement-id'
   },
+  yar: {
+    get: sandbox.stub(),
+    set: sandbox.stub()
+  },
```

Comment:
I don't think you need to stub yar if you're stubbing the `sessionManager` helpers method

Reply:
Yeah - `yar.get` and `yar.set` need to be stubbed because `session-forms` uses those methods and isn't in itself stubbed :D

Reply:
Ah yes, you're right